handler/instance-started.go

```diff
@@ -198,6 +198,7 @@ func (h *InstanceStarted) triggerImportDimensionOptions(ctx context.Context, dim
 	log.Info(ctx, "Triggering dimension-option import process")
 
 	for _, d := range dimensions{
+		log.Info(ctx, "TIRIGERRERER")
```

Comment:
I think this log was meant for debugging only? If that's the case, can you remove it please?

Reply:
lol, looks like it